edge-modules/TwinTester/IResultHandler.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TwinTester
+{
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Shared;
+
+    interface IResultHandler
```

Comment:
What is this interface supposed to do?
Also if it is Twin specific, we should name it so.